lib/ReplaceOpenCLBuiltinPass.cpp

```diff
@@ -1950,6 +1956,22 @@ bool ReplaceOpenCLBuiltinPass::replaceAtomics(Module &M) {
   }
 
   const std::map<const char *, llvm::AtomicRMWInst::BinOp> Map2 = {
+      {"_Z8atom_addPU3AS1Vii", llvm::AtomicRMWInst::Add},
+      {"_Z8atom_addPU3AS1Vjj", llvm::AtomicRMWInst::Add},
+      {"_Z8atom_subPU3AS1Vii", llvm::AtomicRMWInst::Sub},
+      {"_Z8atom_subPU3AS1Vjj", llvm::AtomicRMWInst::Sub},
+      {"_Z9atom_xchgPU3AS1Vii", llvm::AtomicRMWInst::Xchg},
+      {"_Z9atom_xchgPU3AS1Vjj", llvm::AtomicRMWInst::Xchg},
+      {"_Z8atom_minPU3AS1Vii", llvm::AtomicRMWInst::Min},
```

Comment:
One of these should map to AtomicRMWInst::UMin. Probably the `j` version, but I don't remember all the mangling off hand.

Reply:
Yes, you're right, well spotted. The conformance tests probably need to be fixed as well.